backends/core/models.py

```diff
@@ -74,6 +83,21 @@ def recipients(self):
       return self.emails_for_notifications.split()
     return []
 
+  def _add_multi_to_memcache(self, mapping,retries = 10, time = MEMCACHE_DEFAULT_EXPIRATION_TIME):
+    """
+        Set multiple values in memcache, prefixed by the id of the pipeline
+        and having a default expiration time of 24 hours
+    """
+    client = memcache.Client()
+    while retries > 0:
```

Comment:
I would prefer to avoid mutating the parameter directly, change the parameter name to `max_retries`, and then increment a counter locally in the method.

Reply:
done